ru_RU.json

```diff
@@ -0,0 +1,956 @@
+  "DEBUG_GLOBAL_FREEZE_TOGGLE": "Заморозить",
+  "DEBUG_GLOBAL_OFFSET": "Смещение: %d",
+  "DEBUG_GLOBAL_OFFSET_COUNT": "Количество смещений:",
+  "DEBUG_GLOBAL_SIZE": "Размер:",
+  "DEBUG_GLOBALS_ADD": "Добавить глобальный",
+  "DEBUG_GLOBALS_BASE_ADDRESS": "Базовый адрес:",
+  "DEBUG_GLOBALS_ENABLE_FREEZING": "Включить заморозку",
+  "DEBUG_GLOBALS_NAME": "Имя:",
+  "DEBUG_GLOBALS_NEW": "Новый глобальный",
+  "DEBUG_GLOBALS_VALUE": "Ценить:",
+  "DEBUG_LOG_METRICS": "Метрики журнала",
+  "DEBUG_LOG_NATIVE_SCRIPT_HOOKS": "Нативные скриптовые хуки",
+  "DEBUG_LOG_SCRIPT_EVENT": "Включить ведение журнала событий сценария",
+  "DEBUG_LOG_SCRIPT_EVENT_FILTER_BY_PLAYER": "Фильтровать по игроку",
+  "DEBUG_LOG_TREE_SCRIPT_EVENT": "Регистрация событий сценария",
+  "DEBUG_REMOVE_FROM_BAD_SPORT": "Удалить из Bad Sport",
+  "DEBUG_SCRIPT_EVENT_ARG": "Арг[%d]",
+  "DEBUG_SCRIPT_EVENT_ARG_COUNT": "Количество аргументов сценария:",
+  "DEBUG_SCRIPT_EVENT_EVERYONE": "Отправить всем",
+  "DEBUG_SCRIPT_EVENT_PLAYER_ID": "ID игрока:",
+  "DEBUG_SCRIPT_EVENT_SEND_EVENT": "Отправить событие",
+  "DEBUG_SCRIPTS_SUB_TITLE": "Эти скрипты отвечают за все зацикленные функции.\n",
+  "DEBUG_TAB_GLOBALS": "Глобальные",
+  "DEBUG_TAB_LOCALS": "Локальные",
+  "DEBUG_TAB_MISC": "Разное",
+  "DEBUG_TAB_SCRIPT_EVENTS": "События сценария",
+  "DEBUG_TAB_SCRIPTS": "Скрипты",
+  "DEBUG_TABS_LOGS": "Логи",
+  "DEBUG_WINDOW": "Отлаживать",
+  "DECLOAK": "Раскрыть",
+  "DEFAULT": "По умолчанию",
+  "DEFAULT_CMD_PERMISSIONS": "Разрешения команды по умолчанию",
+  "DELETE": "Удалить",
+  "DELETE_TRAIN": "Удалить поезд",
+  "DELETE_TRAIN_DESC": "Вы не можете удалить поезд, находясь в нем.",
+  "DERAIL_TRAIN": "Сойти с рельсов поезд",
+  "DESCENDING": "По убыванию",
+  "DESYNC_KICK": "Рассинхронный кик",
+  "DIRECTION": "Направление",
+  "DISABLE_CEO_MONEY": "Отключить деньги босса",
+  "DISABLE_CEO_MONEY_DESC": "Блокирует выпадение денег босса на протяжении всей сессии. ",
+  "DISABLE_COLLISION": "Отключить столкновение",
+  "DISABLE_ENGINE_AUTO_START": "Отключить автозапуск двигателя",
+  "DISABLE_FILTER": "Отключить фильтр",
+  "DISABLE_PEDS": "Отключить пешеходов",
+  "DISABLE_TRAFFIC": "Отключить трафик",
+  "DMG_MULTIPLR": "Множитель урона",
+  "DONT_STOP": "Не останавливаться",
+  "DRIVE_TRAIN": "Водить поездом",
+  "DRIVER": "Водитель",
+  "DRIVING_STYLE": "Стиль вождения",
+  "DRIVING_STYLE_SET_TO": "Стиль вождения установлен на {}.",
+  "DROWN": "Утопление",
+  "EMERGENCY_STOP": "Экстренная остановка",
+  "ENABLE_SPECIAL_AMMO": "Включить специальные боеприпасы",
+  "ENABLED": "Включено",
+  "END_KICK": "Кик завершения сеанса",
+  "END_KICK_DESC": "Это может занять около 10 секунд, чтобы кикнуть игрока.",
+  "ENTER_INTERIOR": "Войти в интерьер",
+  "ENTER_INTERIOR_DESC": "Входит внутрь игрока",
+  "ERROR_FAILED_TO_TAKE_CONTROL": "Не удалось взять под контроль транспортное средство игрока.",
+  "ERROR_INVALID_VEHICLE": "Подходящий автомобиль не найден.",
```

Comment:
ERROR_FAILED_TO_TAKE_CONTROL and ERROR_INVALID_VEHICLE inconsistently translate the word vehicle, in take control it's more literal, in invalid vehicle it's assumed it's a car.